tests/libs/util/socket_helper.cpp

```diff
@@ -220,7 +220,7 @@ _datagram_client_socket::send_message_to_remote_host(
         1,
         reinterpret_cast<LPDWORD>(&bytes_sent),
         send_flags,
-        (PSOCKADDR)&remote_address,
+        (const PSOCKADDR)&remote_address,
```

Comment:
```suggestion
        (const SOCKADDR*)&remote_address,
```
const PFOO does not mean the same as const FOO*